lib_runtime/mirage_runtime.mli

```diff
@@ -101,8 +111,4 @@ module Hooks : sig
 
   val iter : unit t -> unit
   (** [iter s] calls the hooks registered in [s] in sequence. *)
-
-  val iter_p : unit Lwt.t t -> unit Lwt.t
```

Comment:
If you remove `iter_p` then we need to expose the fact that the hooks are stored in a  `Lwt_dllist` sequence (otherwise you can't do anything with `Hooks.exit`

Reply:
yes, I noticed this. I added a `run_exit_hooks` subsequently -- is this fine with you?